sccm/apps/deploy-use/packages-and-programs.md

```diff
@@ -23,6 +23,9 @@ System Center Configuration Manager continues to support packages and programs t
 - “One-off” scripts that do not need to be continually monitored  
 - Scripts that run on a recurring schedule and cannot use global evaluation
 
+> [!TIP]
+> Beginning with 1706 (pre-release) and 1802 (production), Configuration Manager introduce the ability to [run scripts](/sccm/apps/deploy-use/create-deploy-scripts) from the console.  This may be a better solution for some of the scenarios listed above in place of using the packages and programs feature.
```

Comment:
A few minor changes for style

FYI, I removed the opening clause as the feature is available in all currently supported versions. So no reason to call it out anymore. 😄 

```suggestion
> Consider using the [run scripts](/sccm/apps/deploy-use/create-deploy-scripts) feature in the Configuration Manager console. Scripts may be a better solution for some of the preceding scenarios instead of using packages and programs.
```

Reply:
Noted for future changes, thanks!  I think I broke my fork that had this change, did you update the file in the commit, or should I refork?